bot.js

```diff
@@ -209,16 +214,64 @@ function memberUpdateHandler(oldMember, newMember) {
     }
   }
 }
+// Also run once on startup of script
+// Will post if all conditions are met (time is correct, post is not stickied, post have not been posted)
+const redditInterval = async () => {
+  const shouldPost = isRedditPostingTime();
+  if (!shouldPost) return;
+  try {
+    const res = await Phin({ url: REDDIT_URL, parse: "json" });
+    const postList = res.body.data.children;
+    let validPost = null;
+    let startAt = 0;
+    while (!validPost) {
+      const [postLink, position] = getNthNonStickiedPost(postList, startAt);
+      if (await havePostedAlready(postLink)) {
+        startAt = position + 1;
+        continue;
+      } else {
+        validPost = postLink;
+      }
+    }
+    const channel = await client.channels.get(REDDIT_POSTING_CHANNEL_ID);
+    channel.send(validPost);
+  } catch(error) {
+    console.error(error);
```

Comment:
Add some context here with another console.error above explaining the context. Ie: "Failed to fetch and post Reddit post on interval".